torch/fx/experimental/symbolic_shapes.py

```diff
@@ -1240,6 +1240,10 @@ def method_to_operator(method):
         op = getattr(operator, method_attr)
     return op
 
+def convert_to_symint(symbool: torch.SymBool) -> torch.SymInt:
```

Comment:
Hmm. Under the prevailing naming conventions, we would have called this `sym_int`, but we already have a function named that. Hmm... can we include bool in the name somehow? `cast_symbool_to_symint_guardless` or smthg

Reply:
yeah, cast_symbool_to_symint_guardless is accurate.